calibre.json

```diff
@@ -0,0 +1,64 @@
+{
+  "Calibre": {
+    "description": "Calibre is an open source e-Book management program.<p>Calibre allows for the management of e-Books and syncing across devices using the content server.</p><p>For more information visit <a href='https://calibre-ebook.com/' target='_blank' rel='noopener noreferrer'>Calibre's Official Site</a> and <a href='https://hub.docker.com/r/linuxserver/calibre' target='_blank' rel='noopener noreferrer'>the Docker container page.</a></p>",
+    "more_info": "",
+    "version": "0.0.1",
+    "website": "https://calibre-ebook.com/",
+    "ui": {
+      "slug": ""
+    },
+    "volume_add_support": true,
+    "containers": {
+      "calibre": {
+        "image": "linuxserver/calibre",
+        "launch_order": 1,
+        "opts": [
```

Comment:
No need to define the `-v config` binding as it is defined by the user with the `"volumes"` object below. You should thus be able to delete the `"opts"` object altogether here.